Random test failure - TestLazySeriesIterator_RangeValues

https://circleci.com/gh/weaveworks/cortex/1223
https://circleci.com/gh/weaveworks/cortex/1232

```
--- FAIL: TestLazySeriesIterator_RangeValues (0.00s)

	Error Trace:	iterator_test.go:118

	Error:		Not equal: []model.SamplePair{model.SamplePair{Timestamp:1499948272918, Value:0}} (expected)

			        != []model.SamplePair{} (actual)

			

			Diff:

			--- Expected

			+++ Actual

			@@ -1,3 +1,2 @@

			-([]model.SamplePair) (len=1 cap=1) {

			- (model.SamplePair) 0 @[1499948272.918]

			+([]model.SamplePair) (cap=1) {

			 }
```

--- a/pkg/chunk/iterator_test.go
+++ b/pkg/chunk/iterator_test.go
@@ -3,6 +3,7 @@ package chunk
 import (
 	"context"
 	"testing"
+	"time"
 
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/prometheus/storage/metric"
@@ -106,7 +107,7 @@ func TestLazySeriesIterator_RangeValues(t *testing.T) {
 		}{
 			{
 				iterator:        iterator,
-				interval:        metric.Interval{OldestInclusive: now, NewestInclusive: now},
+				interval:        metric.Interval{OldestInclusive: now.Add(-time.Minute), NewestInclusive: now.Add(time.Minute)},
 				expectedSamples: dummySamples,
 			},
 		} {

